js/host.js

```diff
@@ -80,16 +80,17 @@ function handleFiles()
     fileList.forEach(file =>
     {
         var reader = new FileReader();
-        if (file.name.endsWith(".ck")) {
-            reader.onload = e => {
+        if (file.name.endsWith(".ck")) 
+        {
```

Comment:
Is our convention going to be { } on new lines? I'm cool with it. @terryzfeng @nshaheed @gewang

Reply:
we have been going with { } on new lines in c++ and ck (except when it looks terribad); I'm also good with continuing this convention in js

```
// is this terribad? extra low brackets to content ratio?
if( yes )
{
    x = 1;
}
else
{
    x = 0;
}

// is this less terribad? (more concise; might be error-prone)
if( yes ) x = 1; else x = 0;
// or this? (more concise and readable; less error-prone )
if( yes ) { x = 1; } else { x = 0; }
```

Reply:
+1 for consistency 

Personally, I think the bottom two examples are harder to understand.

What about

```
if (yes) x = 1;
else x = 0;
```

or

```
x = 0;
if (yes) x = 1;
```

Reply:
lol yes agree this is more readable:
```if (yes) x = 1;
else x = 0;```

or if explicitly keeping the { }
```if (yes) { x = 1; }
else { x = 0; }```

depending on much we want to adhere or break from tradition, we have mostly been doing spacing inside parens and no spacing outside, e.g.,

```if( yes ) x = 1;
else x = 0;```

Reply:
Regarding the comments above, I'm down to keep writing new line brackets in js as well. But I do think popularity in js does favor inline brackets down the board, especially in lambda functions
```
foo().then((value) => {
        // stuff
    }); 
```
but this could coexist with control flow brackets
```
if (true)
{
}
```
Regarding adding brackets for one if statements, my general thoughts are always add brackets lol

Reply:
for spaces from parentheses, I've been doing `if (true)` for keywords and `function(params)` for function calls (I think I learned those from Java conventions?

So spacing inside convention is `if( true )` for keywords and then `function(params)`?